main/usbutils/template.py

```diff
@@ -17,4 +17,8 @@
 
 def post_install(self):
     self.rm(self.destdir / "usr/bin/lsusb.py")
-    self.rm(self.destdir / "usr/lib/pkgconfig", recursive=True)
+
+
+@subpackage("usbutils-devel")
+def _devel(self):
+    return self.default_devel()
```

Comment:
this needs a dependency on `{pkgname}={pkgver}-r{pkgrel}` as it will not gain one automatically through a library symlink

Reply:
Added